experimental-examples/tina-cloud-starter/pages/_app.tsx

```diff
@@ -27,7 +27,7 @@ const App = ({ Component, pageProps }) => {
               /**
                * Enables `tina-admin` specific features in the Tina Sidebar
                */
-              cms.flags.set("tina-admin", false);
+              cms.flags.set("tina-admin", true);
```

Comment:
I'm guessing this accidentally slipped through.  We've been keeping `tina-cloud-starter`'s `tina-admin` flag set to `false`.
```suggestion
              cms.flags.set("tina-admin", false);
```